datasets/fhad/make_data.py.amltmp

```diff
@@ -0,0 +1,237 @@
+from dataclasses import dataclass
```

Comment:
This file probably shouldn't be in your PR and `.amltmp` should added to the `.gitignore` after it is removed.